extensions/ql-vscode/src/extension.ts

```diff
@@ -480,6 +480,23 @@ async function activateWithInstalledDistribution(
     }
   }
 
+  async function openReferencedFile(
+    selectedQuery: Uri
+  ): Promise<void> {
+    if (qs !== undefined) {
+      if (await cliServer.supportsResolveQlref()) {
+        const resolved = await cliServer.resolveQlref(selectedQuery.path);
+        const uri = Uri.file(resolved);
+        await window.showTextDocument(uri, { preview: false });
+      } else {
+        helpers.showAndLogErrorMessage(
+          'Jumping from a .qlref file to the .ql file it references is not '
+          + 'supported with the CLI version you are running.\n'
+          + 'Please upgrade your CLI to use this feature.');
```

Comment:
Not necessary, but it would be neat if you could add a link here that can automatically run the check for updates command when run. Not even sure if this is possible.

Reply:
I agree this would be nice, but as far as I can tell it's a little tricky in the current setup since the functions we need to call to do this live in the code that activates the extension and are no longer in scope at this point. I think we should refactor so that there is a function for updating the CLI inside `DistributionManager` that we can call after the extension has been activated, but this seems out of the scope of this PR.